lib/macro/macro.hpp

```diff
@@ -168,6 +170,7 @@ class Macro {
 	bool _conditionSateChanged = false;
 	bool _performActionsOnChange = true;
 	bool _skipExecOnStart = false;
+    bool _stopActionsIfNotDone = false;
```

Comment:
Can you please rebase on the current master?
I think I finally fixed the issue that prevented clang-format form doing its job in the CI runs and thus spotting formatting issues.

Reply:
Sure